src/electionguard/directory.c

```diff
@@ -67,4 +67,15 @@ bool create_directory(const char *path)
     }   
 
     return true;
+}
+
+bool Directory_exists(const char *path)
+{
+    struct stat stats = {0};
+
+    if (stat(path, &stats) == 0 && S_ISDIR(stats.st_mode)) 
+    {
+        return true;
```

Comment:
Multiple return values for a very simple function. Not a bug, but it is not the best practice.